### ISSUE TITLE
make AI use curia coffers - in a balanced manner, we don't want the reformation to spawn in 1700

### ISSUE THREAD

**Issue by Mxt-dot** [Mxt-dot/BetterAI#29]

**Comment by Unoffe** [comment]
Is there any problems? I never saw situations when reformation spawns this late.

**Comment by Mxt-dot** [comment]
You are right, there's currently no issue per se. The AI very rarely uses the curia coffers which I want to change. I want the AI to finally utilize the money that's spent.

The potential issue is that with that huge amount of money the reformation desire can be pushed further back. I saw situations where you could push it back to 1700, which would be the case if I optimize the AI to utilize it completely. Therefore I will have to figure out the best way to balance it out :D
